sql/core/src/test/scala/org/apache/spark/sql/execution/python/BatchEvalPythonExecSuite.scala

```diff
@@ -97,6 +100,17 @@ class BatchEvalPythonExecSuite extends SparkPlanTest with SharedSQLContext {
     }
     assert(qualifiedPlanNodes.size == 1)
   }
+
+  test("Python UDF refers to the attributes from more than one child in join condition") {
```

Comment:
I would add a JIRA prefix here - this change sounds more like fixing a particular problem.

Reply:
Got it, will add in this commit.